src/jvmCucumberTest/resources/requirements/ORD3.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Knowledge Base management
+
+  @single
+  Scenario: Name of current Knowledge Base should be displayed
+    Given I start the client application
+    Then the displayed KB name should be Thyroids
+    And stop the client application
+  Scenario: A previously exported Knowledge Base can be imported
+    Given I start the client application
```

Comment:
It's a bit pedantic, but I think it's clearer to stick to the Given/When/Then format, e.g.

Given I start the client application
    And the displayed KB name is Thyroids
    When I import the configured zipped Knowledge Base Whatever
    Then the displayed KB name should be Whatever